ui/src/app/kvm/views/KVMDetails/KVMSummary/KVMSummary.tsx

```diff
@@ -27,8 +27,18 @@ const KVMSummary = (): JSX.Element => {
   }, [dispatch]);
 
   if (!!pod) {
+    // Fetch the IP address that is contained in the power_address. This is the
+    // only place the IP address is exposed.
+    const ip_address = pod.power_address.match(/[\d.]+/)[0];
     return (
       <>
+        <div className="u-flex">
+          <p className="u-nudge-left">LXD URL:</p>
```

Comment:
This text depends on the pod type. In the legacy version, virsh pods = "Virsh:" lxd pods = "LXD URL:"